src/lib/storageRouter.ts

```diff
@@ -26,17 +26,29 @@ function useStorageRouterStore() {
     [activeStorageId, push]
   )
 
-  const navigateToNote = useCallback(
-    (storageId: string, noteId: string, noteFolderPathname = '/') => {
+  const navigateToNoteWithEditorFocus = useCallback(
+    (
+      storageId: string,
+      noteId: string,
+      noteFolderPathname = '/',
+      focusLineNum = 0
+    ) => {
       push(
         `/app/storages/${storageId}/notes${
           noteFolderPathname === '/' ? '' : noteFolderPathname
-        }/${noteId}`
+        }/${noteId}/${focusLineNum}`
```

Comment:
In my opinion, the line information should be presented to hash.`.../${nodeId}/#L${focusLineNum}`

Reply:
Fixed to hash, I added "#L${focusLine},${focusCol}"
should be correct now.